sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -550,6 +557,7 @@ export interface NotificationHubsResponse {
 // @public
 export class NotificationHubsServiceClient {
     constructor(connectionString: string, hubName: string, options?: NotificationHubsClientOptions);
+    beginSubmitNotificationHubJob(notificationHubJob: NotificationHubJob, options?: OperationOptions): Promise<SimplePollerLike<OperationState<NotificationHubJob>, NotificationHubJob>>;
```

Comment:
We discussed offline adding an `updateIntervalInMs` option to let the customer control the polling interval but that is an additive change that can be added later.

Reply:
Added via `PolledOperationOptions`